src/libsyntax_pos/span_encoding.rs

```diff
@@ -31,11 +31,13 @@ pub struct Span(u32);
 
```

Comment:
We should be able to remove the #[repr(packed)] here and derive copy/clone/partialeq/hash/etc